docs/_sidebar.md

```diff
@@ -1,26 +1,25 @@
-**About TechPass**
-- [Overview](home)
-- [What's new](/whats-new/whats-new.md)
-**Getting started**  
-- [Onboard to TechPass](onboard-to-techpass.md)
-  - [Sign up and onboard](sign-up-and-onboard-to-techpass)
-  - [Get invited and onboard](get-invited-and-onboard-to-techpass)
-**Verify TechPass login** 
-- [Log in with TechPass](log-in-with-techpass)
-**TechPass Portal**
-- [Request for SEED provisioning](request-for-seed-provisioning)
-- [Register Intune device ID](register-intune-device-id)
-- [Edit TechPass profile](edit-profile)
-**Reset MFA**
-- [WOG account](reset-security-verification-for-wog-account)
-- [TechPass account](reset-techpass-mfa-for-new-device)  
-**Manage password**
-- [Reset password](reset-password)
-**TechPass support resources**
-- [MFA fraud alert](mfa-fraud-alert)
-- [TechPass status](techpass-status)
-- [FAQs](/support/overview.md)
-- [Terms and policies](terms-and-policies)
-- [Glossary](glossary)
+- **About TechPass**
+  - [Overview](home)
+  - [What's new](/whats-new/whats-new.md)
+- **Getting started**
+  - [Onboard to TechPass](onboard-to-techpass.md)  
+    - [Sign up and onboard](sign-up-and-onboard-to-techpass)
+    - [Get invited and onboard](get-invited-and-onboard-to-techpass)
+- **Verify TechPass login** 
+  - [Log in with TechPass](log-in-with-techpass)
+- **TechPass Portal**
+  - [Request for SEED provisioning](request-for-seed-provisioning)
+  - [Register Intune device ID](register-intune-device-id)
+  - [Edit TechPass profile](edit-profile)
+- **Reset MFA**
+  - [WOG account](reset-security-verification-for-wog-account)
+  - [TechPass account](reset-techpass-mfa-for-new-device)  
```

Comment:
"WOG account" and "TechPass account" would the user know, since they only know if they signed up or were invited?

Reply:
addressed this.